crates/transaction-pool/src/validate/eth.rs

```diff
@@ -1067,15 +1068,10 @@ impl<Client> EthTransactionValidatorBuilder<Client> {
             max_tx_input_bytes,
             max_tx_gas_limit,
             disable_balance_check,
-            ..
+            max_blob_count,
+            additional_tasks: _,
```

Comment:
i see, with this we make use max_blob_count based on the parameters active at the timestamp of the latest block